Makefile

```diff
@@ -500,16 +508,27 @@ KBUILD_CFLAGS   := -Wall -Wundef -Werror=strict-prototypes -Wno-trigraphs \
 		   -Wno-format-security \
 		   -std=gnu89
 KBUILD_CPPFLAGS := -D__KERNEL__
+KBUILD_RUSTCFLAGS :=
+# TODO: a simple way to update `Cargo.lock` when we add a new driver
+# TODO: another option is using explicit target specs, e.g.
+# `--target=$(srctree)/arch/$(SRCARCH)/rust-target-spec.json`
+KBUILD_CARGOFLAGS := $(CARGO_VERBOSE) --locked \
+		    -Z build-std=core,alloc -Z unstable-options \
+		    --out-dir=out --target=x86_64-linux-kernel
```

Comment:
This probably needs to come from the ARCH somehow. Maybe not a blocker yet though?

Reply:
Not a blocker, indeed. My original approach was to put the spec as shown in the comment above (we'd still need to differentiate between sub-arche).